playwright/models/Layout.ts

```diff
@@ -24,7 +24,7 @@ export class Layout {
   constructor(page: Page) {
     this.page = page;
 
-    // portfolio
+    // portfolio && accounts
     this.totalBalance = page.locator('data-test-id=total-balance');
 
     // drawer
```

Comment:
For future we should probably move Drawer into its own PO

Reply:
Agreed, we'll do it step by step 👍